### ISSUE TITLE
Units formatting tests fail on windows

### ISSUE THREAD

**Issue by MichaelClerx** [ModellingWebLab/cellmlmanip#233]
```
============================= test session starts =============================
platform win32 -- Python 3.6.8, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: C:\projects\cellmlmanip
plugins: cov-2.8.1
collected 139 items
tests\test_model_evaluation.py .                                         [  0%]
tests\test_model_functions.py ...........................                [ 20%]
tests\test_parser.py .......s..........                                  [ 33%]
tests\test_printer.py ............                                       [ 41%]
tests\test_rdf.py ...........                                            [ 49%]
tests\test_transpiler.py .............................................   [ 82%]
tests\test_unit_conversion.py ..............                             [ 92%]
tests\test_units.py .F.F.......                                          [100%]
================================== FAILURES ===================================
________________________ TestUnits.test_add_base_unit _________________________
self = <tests.test_units.TestUnits object at 0x000000795EEC6860>
    def test_add_base_unit(self):
        """Tests UnitStore.add_base_unit()."""
    
        unitstore = UnitStore()
        assert not unitstore.is_defined('uu')
        uu = unitstore.add_base_unit('uu')
        assert unitstore.is_defined('uu')
>       assert unitstore.format(uu, True) == '1.0 uu'
E       AssertionError: assert '1 uu' == '1.0 uu'
E         - 1 uu
E         + 1.0 uu
E         ?  ++
tests\test_units.py:62: AssertionError
____________________________ TestUnits.test_format ____________________________
self = <tests.test_units.TestUnits object at 0x000000795ED46B38>
    def test_format(self):
        """Tests formatting of units."""
    
        # Test basic formatting and base unit expansion
        store = UnitStore()
        store.add_unit('x', 'second')
        y = store.add_base_unit('y')
        z = store.add_unit('z', 'y * meter / x ** 2')
        assert store.format(z) == 'z'
        assert store.format(z, True) == '1.0 meter * y / second ** 2'
    
        # Test if internal name manipulation can mess things up
        a = store.add_base_unit(str(y))
        b = store.add_unit(str(z), '2 * z /' + str(y))
        assert store.format(a) == str(y)
        assert store.format(b) == str(z)
>       assert store.format(a, True) == '1.0 ' + str(y)
E       AssertionError: assert '1 store50_y' == '1.0 store50_y'
E         - 1 store50_y
E         + 1.0 store50_y
E         ?  ++
tests\test_units.py:97: AssertionError
================== 2 failed, 136 passed, 1 skipped in 11.52s ==================
```

**Comment by jonc125** [comment]
😱

**Comment by MichaelClerx** [comment]
Fixed in #238
